tools/st2_deploy.sh

```diff
@@ -97,6 +99,22 @@ else
     MISTRAL_STABLE_BRANCH="st2-0.5.1"
 fi
 
+update_hosts_file()
+{
+  if [ -z "`cat /etc/hosts | grep -e '127.0.0.1\slocalhost'`" ]; then
```

Comment:
It wasn't always this way.  I don't know when it changed but in the beginning it wasn't happening.